Makefile

```diff
@@ -70,11 +72,14 @@ install: $(BIN)/$(EXECUTABLE)
 	cp $< $(GOPATH)/bin/
 
 .PHONY: build
-build: $(BIN)/$(EXECUTABLE)
+build: $(BIN)/$(EXECUTABLE) templates/.VERSION
 
 $(BIN)/$(EXECUTABLE): $(wildcard *.go)
 	go build -v -tags '$(TAGS)' -ldflags '-s -w $(LDFLAGS)' -o $@
 
+templates/.VERSION:
```

Comment:
IMHO we should drop this version file. People that build from source should update the templates on their own.